packages/kbn-ftr-common-functional-services/services/saml_auth/serverless/auth_provider.ts

```diff
@@ -34,8 +34,18 @@ const getDefaultServerlessRole = (projectType: string) => {
   }
 };
 
+const isRoleManagementExplicitlyEnabled = (args: string[]): boolean => {
+  const roleManagementArg = args.find((arg) =>
+    arg.startsWith('--xpack.security.roleManagementEnabled=')
+  );
+
+  // Return true if the value is explicitly set to 'true', otherwise false
+  return roleManagementArg?.split('=')[1] === 'true' || false;
```

Comment:
Why is `true` a string literal, but false is a boolean?

Reply:
I get that coercion will make true become a boolean, but that's if coercion is used.

Reply:
we are looking for the exact `--xpack.security.roleManagementEnabled=true` match in Kibana server arguments to understand if roles management was explicitly enabled. I believe Kibana won't start if you try to pass this flag with something other than `true` or `false`

Reply:
I mean only 'true' is expected to enable the functionality, so I set flag to `false` for any other case (including argument is completely missing)

Reply:
I don't think you can start Kibana with `--xpack.security.roleManagementEnabled='true'`, schema validation won't allow it. I don't try validate the cases for Kibana server, I believe FTR won't event start if literal 'true' or 'false' will be in arguments